src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -402,6 +416,33 @@ def _verify_app_source_dir_impl(src_dir, temp_dir, mode, enforce=True):
         logging.warn('%s contained syntax errors, see above for details' % (files_str,))
 
 
+def _get_all_cluster_bootstrap_script_names(manifest):
+    script_names = []
+
+    if 'systemRequirements' in manifest['runSpec']:
```

Comment:
If we throw that error here with no other changes, it will appear for _any_ attempt to build an app that specifies `runSpec.systemRequirements`. Do we want that?

Reply:
I meant we could only throw an error when a bootstrapping config is defined under `runSpec.systemRequirements` and only allow it in `regionalOptions`. When we `dx get` an app/applet we also don't inline this into dxapp.json any more - the `systemRequirements` is only under `regionalOptions`.